ferrocene/doc/user-manual/src/targets/thumbv7em-none-eabi.rst

```diff
@@ -49,3 +49,20 @@ To use the target, the following additional flags must be provided to
 ``rustc``:
 
 * ``--target=thumbv7em-none-eabi``
+
+Testing Facade (Experimental)
+-----------------------------
+
+Bare metal targets cannot use test harnesses that require ``libc``. Instead, a testing facade can
+be used instead that emulates the bare metal target, but also provides access to ``std``
+functionality.
+
+The following additional archive is needed when :doc:`installing </rustc/install>`:
+
+* ``rust-std-thumbv7em-ferrocene.facade-eabi``
+
+This target is the same as the one it proxies, except it includes a Linux ``libc``,
+this means it can use ``std`` for testing and enriched interactive development on a
```

Comment:
```suggestion
This target is the same as the one it proxies, except it includes a Linux ``libc``,
which means it can use ``std`` for testing and enriched interactive development on a
```